tests/python/gaia-ui-tests/gaiatest/apps/clock/app.py

```diff
@@ -27,9 +24,20 @@ def launch(self):
             Wait(self.marionette).until(expected.element_present(
                 *self._alarm_create_new_locator))))
 
-    @property
-    def alarms(self):
-        return [self.Alarm(self.marionette, alarm) for alarm in self.marionette.find_elements(*self._all_alarms_locator)]
+    def switch_view(self, view_name):
+        Wait(self.marionette).until(
+            expected.element_present(*(By.ID, self._clock_views.get(view_name)))).tap()
+        if view_name == 'stopwatch':
+            from gaiatest.apps.clock.regions.stopwatch import StopWatch
+            return StopWatch(self.marionette)
+        elif view_name == 'timer':
+            from gaiatest.apps.clock.regions.timer import Timer
+            return Timer(self.marionette)
+        elif view_name == 'alarm':
+            from gaiatest.apps.clock.regions.alarm import Alarm
+            return Alarm(self.marionette)
+        else:
+            raise AttributeError('{} is not a view that you can switch to'.format(view_name))
 
     @property
     def banner_notification(self):
```

Comment:
After taking a closer look at the app, I think the next 2 functions should belong to the Alarm view. What do you think @silne30 ?